src/miner.cpp

```diff
@@ -298,6 +311,43 @@ void BlockAssembler::SortForBlock(const CTxMemPool::setEntries& package, std::ve
     std::sort(sortedEntries.begin(), sortedEntries.end(), CompareTxIterByAncestorCount());
 }
 
+CFeeRate BlockAssembler::minTxFeeRate()
+{
+    int packages_selected = 0;
+    int descendants_updated = 0;
+    CFeeRate min_fee_rate = CFeeRate(COIN, 1);
+
+    resetBlock();
+    pblocktemplate.reset(new CBlockTemplate());
+
+    if (!pblocktemplate.get()) {
```

Comment:
This seems impossible to hit. You've just constructed a new `std::unique_ptr<CBlockTemplate>`.

Reply:
yeah, looks like I copied this from `CreateNewBlock` and thought it was a "safety check" but I agree, it seems pointless. Removed.